Gems/ROS2/Code/Include/ROS2/Gripper/GripperRequestBus.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+
+namespace ROS2
+{
+    //! The interface allows to control gripper components through GripperCommand actions.
+    //! It is a bus that allows communication between ROS2 GripperCommand Action server with the particular implementation of the gripper.
+    //! It encapsulates GripperCommand commands and getters that allow to build the feedback and result messages.
+    //! @see <a href="https://github.com/ros-controls/control_msgs/blob/humble/control_msgs/action/GripperCommand.action">GripperCommand</a>
+    class GripperRequests : public AZ::EBusTraits
+    {
+    public:
+        using BusIdType = AZ::EntityId;
+        static constexpr AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+        static constexpr AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
+
+        //! Send new command to the gripper.
+        //! @param position position of the gripper (as a gap size in meters) to be set, for vacuum/electromagnetic gripper it is 0 or 1.
+        //! @param maxEffort maximum effort of the gripper to be set in Newtons.
+        //! @return Nothing on success, error message if failed.
+        virtual AZ::Outcome<void, AZStd::string> GripperCommand(float position, float maxEffort) = 0;
```

Comment:
Would it be useful to return the result for failure similar to the change introduced to Manipulator Trajectory API in https://github.com/o3de/o3de-extras/pull/386/files? It would be best to update error handling as per indicated PR.

Reply:
The GripperCommand result message doesn't support errors, so an analogous mechanism wouldn't work (the command is so simple that nothing can really go wrong).
Moreover, I think this class is supposed to be independent of ROS and is supposed to be able to work independent of the ROS interface.